R-package/tests/testthat/test_parameters.R

```diff
@@ -48,7 +48,7 @@ context("parameter aliases")
 
 test_that(".PARAMETER_ALIASES() returns a named list of character vectors, where names are unique", {
   param_aliases <- .PARAMETER_ALIASES()
-  expect_true(is.list(param_aliases))
+  expect_true(identical(class(param_aliases), "list"))
```

Comment:
```suggestion
  expect_identical(class(param_aliases), "list")
```
For the consistency with other checks.

Reply:
ok yeah good catch